src/App.tsx

```diff
@@ -1,52 +1,12 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
-
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
+import { Navigation } from './components/Navigation';
+import { Rout } from './components/Rout';
```

Comment:
It seems like there might be a typographical error in the import statement. The component should be 'Route', not 'Rout'.